Code/Editor/CryEditDoc.cpp

```diff
@@ -1922,29 +1914,26 @@ void CCryEditDoc::LogLoadTime(int time)
     SetFileAttributes(filename.toUtf8().data(), FILE_ATTRIBUTE_ARCHIVE);
 #endif
 
-    FILE* file = nullptr;
-    azfopen(&file, filename.toUtf8().data(), "at");
-
-    if (file)
+    QFile file(filename);
+    if (!file.open(QFile::Append | QFile::Text))
```

Comment:
+1 - I like this transformation to use the Qt file API as opposed to the legacy C API but again as this is something that's most likely currently working as-is (even if it's a bit ugly) we really need a way to test things working before and afterwards. This is less worrying than the SaveDocument change but still would be useful (we might be able to extract this to an object or function we could test in isolation) or at the very least for this code can you verify the output remains identical after running the editor and include that in the review?

Reply:
To be honest, I didn't dig into the current testing framework, and from what I'm seeing there are no tests for `CCryEditDoc` at all. I'd have to find a chunk of time to build at least a basic set of those.

Reply:
Hmm that's cool, as @lawsonamzn has had a poke around too I'm less worried, just something to keep in mind. It's one of those things that gets easier the more we do it (it would be cool if have had support for Approval Tests that make these sorts of legacy tests easier to write - maybe someone in future might add support for it! 🙂)